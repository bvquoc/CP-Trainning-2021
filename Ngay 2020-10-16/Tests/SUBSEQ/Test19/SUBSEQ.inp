800
124 102 213 158 86 82 20 220 234 231 57 110 74 224 59 211 218 172 206 206 220 174 17 216 38 10 84 107 19 163 165 194 84 152 19 6 212 104 232 58 221 60 215 99 248 225 221 248 229 176 76 7 59 84 57 91 186 89 155 196 5 26 17 7 236 34 167 69 111 186 52 129 191 114 27 149 69 23 199 14 98 86 13 49 168 124 8 183 194 209 42 218 8 115 230 103 192 188 237 73 222 106 157 172 213 27 55 212 231 175 79 100 128 145 187 57 29 231 161 86 160 175 35 85 152 85 224 114 221 29 112 39 248 146 193 245 116 122 38 12 199 177 84 151 162 246 107 187 34 167 183 249 23 41 229 226 206 12 72 31 107 108 174 93 78 36 170 4 100 128 173 29 33 107 63 45 231 60 250 162 173 247 203 106 12 106 29 99 230 12 64 28 120 77 78 73 246 230 4 26 18 1 216 81 3 103 227 180 32 27 233 133 135 108 58 44 69 81 88 72 180 75 164 200 168 2 142 109 126 236 154 183 144 171 56 54 72 45 246 210 99 84 16 152 32 52 212 13 197 84 165 62 110 81 98 72 73 93 91 45 2 102 208 182 2 143 53 213 77 25 215 193 98 107 134 183 166 4 48 182 54 218 84 144 94 17 188 198 42 137 205 169 153 20 137 231 41 83 238 33 102 163 15 247 247 69 95 98 124 150 14 92 113 31 51 106 67 5 32 174 166 235 51 143 7 224 50 78 143 174 119 234 177 124 248 86 239 230 132 47 173 59 240 126 224 95 225 201 174 172 166 25 25 188 115 250 197 221 76 163 80 117 12 144 170 44 16 218 121 23 155 118 74 182 68 87 99 89 99 134 219 88 71 83 89 72 43 190 56 204 147 183 148 12 221 78 76 32 89 210 246 227 119 136 193 184 249 24 46 68 73 238 174 6 113 128 150 59 160 10 160 158 26 43 239 204 145 203 206 89 133 62 223 37 224 116 70 154 121 141 249 99 176 176 32 198 232 110 230 5 100 247 240 192 176 30 239 57 104 151 142 154 71 104 42 42 187 62 10 76 172 215 184 124 225 59 228 218 153 215 141 4 153 133 45 194 70 149 172 149 28 185 214 239 159 226 66 10 216 113 105 127 159 178 61 8 56 84 44 248 14 40 179 200 84 34 72 118 167 165 50 239 150 18 134 218 41 221 44 136 233 150 233 241 178 143 114 88 184 222 134 228 204 159 25 178 156 112 71 92 78 112 110 21 100 226 245 21 188 52 91 244 238 54 247 228 213 215 191 199 66 35 70 97 96 103 148 185 214 127 237 77 142 222 223 215 207 157 23 192 13 89 182 26 164 58 137 199 97 23 81 47 12 219 58 141 84 212 178 184 22 248 10 150 77 161 59 213 176 45 161 155 241 54 247 221 167 247 38 197 195 88 130 17 156 216 172 187 43 153 37 95 87 29 228 129 2 233 142 6 58 34 86 170 82 67 53 247 48 205 117 56 29 243 24 26 147 173 101 1 139 130 201 149 90 130 176 91 173 201 87 231 27 222 208 70 202 43 199 210 6 206 153 249 81 114 223 205 53 42 97 15 7 126 127 211 24 139 78 102 69 233 92 14 230 117 203 61 109 3 226 46 112 177 78 238 244 85 42 157 156 32 243 21 10 114 104 116 165 109 5 192 207 9 9 95 34 188 218 125 40 232 192 37 140 47 53 48 127 231 200 161 195 113 31 151 62 37 46 4 218 33 52 79 1 225 129 196 113 176 111 77 106 225 216 99 242 112 78 121 82 177 40 88 90 112 160 3 22 246 