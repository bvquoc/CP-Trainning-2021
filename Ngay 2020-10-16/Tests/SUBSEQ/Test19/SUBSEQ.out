232
3 8 9 10 14 16 17 19 20 21 24 32 37 39 41 43 45 46 47 48 49 57 60 65 70 73 79 88 89 90 92 95 97 98 99 101 105 108 109 115 118 127 129 133 135 136 141 146 148 151 152 155 156 157 177 182 183 189 197 198 203 207 208 211 221 230 232 239 240 247 249 263 264 268 271 272 276 280 282 287 288 291 296 299 304 305 322 326 332 335 337 338 343 345 347 348 354 356 357 358 368 374 381 388 390 392 395 400 401 402 405 406 407 412 425 426 428 429 433 435 441 446 447 449 452 453 454 457 467 472 473 475 477 478 480 486 492 493 494 496 499 510 513 514 522 526 528 531 533 534 535 539 540 542 543 546 556 557 559 562 563 565 566 567 568 569 570 578 579 581 584 585 586 587 590 593 598 604 608 609 610 612 618 623 625 626 628 630 631 636 638 645 648 658 660 664 673 680 682 684 685 687 689 690 692 694 697 698 706 712 715 717 721 726 727 733 742 743 748 749 752 753 760 761 763 771 776 778 784 785 787 800 